### ISSUE TITLE
Transcription Link Needs Existing <a> Tag

### ISSUE THREAD

**Issue by yee-luna-c** [alexanderlewis99/TinyMCEMod#3]
As it stands, using the `Transcription Link` option under Formatting requires that the specified text already have a link `<a href="#">Show Transcription</a>`.

![image](https://user-images.githubusercontent.com/43069771/56164893-48f18900-5f97-11e9-8312-f2c295b7cf7f.png)
*Above: Step 1 of adding a Transcription Link. Below: Step 2 of adding a Transcription Link.*
![image](https://user-images.githubusercontent.com/43069771/56164943-60307680-5f97-11e9-8976-2cbc657317eb.png)

This is rather cumbersome for the user, as it requires an additional step that is not particularly intuitive. Is it possible to have the `Transcription Link` formatting automatically apply the link as well as the class?

**Comment by alexanderlewis99** [comment]
To address this issue, I added a _Add Transcription Link_ button that will create a transcription link with the text "Show Transcription". It inserts the following code when clicked:  `<p><a class="show-transcription" href="#">Show Transcription</a></p>`. Also, I removed the Transcription Link formatting option so users won't be confused by having multiple ways of creating Transcription Links. Let me know if this is a satisfactory solution. Thanks.
